### PR TITLE
[Doc Improvement][Updated TTK develop your apps][3106090]

### PR DESCRIPTION
**Pull request by v-bvishnu**
Replaced with new screenshots

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/76127f72838580627ba672181a48fdcdb18305ab/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdevelop-your-apps-with-teams-toolkit.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cae45cc8-8ded-4a04-a4a0-c7ae2e8947c9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/76127f72838580627ba672181a48fdcdb18305ab/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e6031d4-3157-4cab-9d10-adfdb278d2be) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/38913edf44862f4f70282320ed6eb50350d1870f/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdevelop-your-apps-with-teams-toolkit.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7bd5535a-12fa-463b-b2ec-a5b93f193a94) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/38913edf44862f4f70282320ed6eb50350d1870f/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ce8022d-5080-4e90-89c5-b684bc601e1e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/33133b7840547b5b05394d74f280dc49cd82a081/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdevelop-your-apps-with-teams-toolkit.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c2082f1f-38b7-4c0a-b6ff-82dbf187461c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/33133b7840547b5b05394d74f280dc49cd82a081/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b5abbe4-7b64-457f-b72c-5adf281c392a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c1edc7d24811e564b7e742976e03833f906075e/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdevelop-your-apps-with-teams-toolkit.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b1351e6-f5a7-4492-9f56-86b13be4e461) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c1edc7d24811e564b7e742976e03833f906075e/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/212fe07c-a825-4e1f-9e66-e558829a1bb2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d9cefe93b24629a0c4fdfb209dabf227ea38ed3/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdevelop-your-apps-with-teams-toolkit.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ea9673af-57f2-4de2-83e5-1b910460bcf9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d9cefe93b24629a0c4fdfb209dabf227ea38ed3/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45acf8d1-d1b5-46d6-958e-451ebad98bc7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ You can open your project in Visual Studio and scaffold your project in Visual S
 
 Following are the steps to open your app in Visual Studio:
 
-1. Select **Develop (Preview)** > **Open in Teams Toolkit** > **Open in Visual Studio (.NET)**.
+1. Select **Develop** > **Open in Teams Toolkit** > **Open in Visual Studio (.NET) (Preview)**.
```
[the remark this answers is not in the document]
Comment (v-bvishnu) [reply msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md:70]: Updated.

**Review by summzhan — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e60551a385fe1ff46254f7872c64fc4e7e0c1f5b/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdevelop-your-apps-with-teams-toolkit.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/04536c09-834a-4e29-8da0-9c55c7b92a80) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e60551a385fe1ff46254f7872c64fc4e7e0c1f5b/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a01a6565-f20e-416d-8f15-276687caaa2f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**